spinta/manifests/tabular/helpers.py

```diff
@@ -436,10 +436,15 @@ def read(self, row: Dict[str, str]) -> None:
             row['model'],
         )
 
-        if "/:" in name:
-            name, features = name.rsplit("/:", 1)
-        else:
-            features = None
+        # Check for partial model syntax
+        features = None
+
+        if "/:" in name or "?" in name:
+            if "/:" in name:
+                features = name.rsplit("/:", 1)[1]
+            elif "?" in name:
+                features = "?" + name.split("?", 1)[1]
```

Comment:
```suggestion
                params = "?" + name.split("?", 1)[1]
```

Should be `params` or `geven_params`.